File: test/fixtures/stdlib/string/transform/substring.expected.py
(lambda _: kAssert("hello"[0:0 + 2] == "he"))(None)
(lambda _: kAssert("hello"[1:1 + 3] == "ell"))(None)
(lambda _: kAssert("hello world"[6:6 + 5] == "world"))(None)
(lambda _: kAssert("hello"[2:2 + 100] == "llo"))(None)

File: test/fixtures/stdlib/string/substring.expected.js
(function(_) { return (function() { if (!("hello".substring(0, 0 + 2) == "he")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("hello".substring(1, 1 + 3) == "ell")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("hello world".substring(6, 6 + 5) == "world")) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("hello".substring(2, 2 + 100) == "llo")) throw new Error("Assertion failed"); return true; })(); })(null);
